src/datatable.js

```diff
@@ -195,7 +191,7 @@ export class DataTable {
       labels.push({
         nested: cleanedColumn.indexOf('.') !== -1,
         column: cleanedColumn,
-        label : ucfirst(clean(aliased[1] || aliased[0]))
+        label : clean(aliased[1] || aliased[0])
```

Comment:
If you want to use ucfirst, you need to alias it. 

I need this because i am translating the same key twice, `name` and `Name`. (aurelia-form don't ucfirst it's labels)